hydra-node/src/Hydra/API/Server.hs

```diff
@@ -6,11 +6,12 @@ module Hydra.API.Server (
   APIServerLog,
 ) where
 
-import Hydra.Prelude
+import Hydra.Prelude hiding (TVar, readTVar)
```

Comment:
Using `TVar` from `stm` instead of `io-sim-classes` here because the server already relies on it.

Reply:
Something to improve for the future.

Reply:
Thing is, there's no TChan in io-sim, hence why we went for `stm` here. Seemed like the team wasn't necessarily _against_ us introducing TChan there, but we would have to do the PR :upside_down_face: